src/main/java/com/example/application/backend/utils/images/ImageManager.java

```diff
@@ -46,16 +46,18 @@
 
     private PageEntity pageEntity;
     private UserEntity userEntity;
+    private int role;
 
-    private final String RESOURCES_DIR = "~/uploads";
+    private final String RESOURCES_DIR = "src/main/resources/META-INF/resources/images/images1611532597562-Nordlicht_AmTag.jpg";
```

Comment:
Do not change the path to a repository **in** the project repository. We won't save uploaded files directly in the project tree. Uploaded files should always be saved directly on the server/database. Saving them now in the project would work, but later (when we deploy the app) not anymore, as the code **won't** be rebuilt/recompiled.

Reply:
Das kam glaube ich automatisch als wir probiert hatten ein Bild hochzuladen..

Reply:
ja da wir ja etwas beim ImageManger ändern mussten hat er das einfach mit übernommen wir hätten daran denken müssen uploads wieder rein zu schreiben aber in der Images und PDF mussten wir die rolle hinzufügen damit man es je nach Role sieht oder nicht sieht

Reply:
man kann einzelne Zeilen in intelliJ zurücksetzen